browser/scripts/worldEditor/worldEditor.js

```diff
@@ -257,7 +263,7 @@ WorldEditor.prototype.setSelection = function(selected) {
 	if (!anySelected)
 		this.clearSelection()
 
-	E2.ui.emit('worldEditor:selectionSet')
+	this.emit('selectionSet', this.getSelectedNodes())
```

Comment:
should this not be a Flux dispatch? so that we could be sure this doesn't ever cause any cascading changes.
the UI state store should listen to the dispatch, set the selection state, and emit the selection change.